.cursor/skills/rosa-hcp-provision/resize-gpu-pool-2xlarge.sh

```
@@ -0,0 +1,57 @@
+    --instance-type "$INSTANCE_TYPE" --replicas 1 --subnet "$PRIVATE_SUBNET" --yes
+else
+  echo "ERROR: could not determine state of pool $NEW_POOL:" >&2
+  echo "$describe_output" >&2
+  exit 1
+fi
+
+echo "==> Wait for new GPU node Ready, THEN wait for the driver to actually be usable on it"
+echo "    (node Ready alone does not mean the GPU is schedulable yet):"
+echo "    oc get nodes -l node.kubernetes.io/instance-type=$INSTANCE_TYPE -w"
+echo "    oc wait --for=condition=Ready pod -l app=nvidia-driver-daemonset -n nvidia-gpu-operator --timeout=1800s"
+echo "    oc get node -l nvidia.com/gpu.present=true -o jsonpath='{.items[0].status.allocatable.nvidia\.com/gpu}{\"\n\"}'  # expect 1"
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)